config/datanode.example.toml

```diff
@@ -43,9 +43,10 @@ sync_write = false
 [storage]
 # The working home directory.
 data_home = "/tmp/greptimedb/"
+# global_store = "File"
+# Storage type.
+[[storage.store]]
```

Comment:
This is a breaking change. `storage.store` is a little odd. We'd better find out a new name or revisit the storage config later.